ts/nni_manager/training_service/reusable/environments/remoteEnvironmentService.ts

```diff
@@ -208,6 +208,27 @@ export class RemoteEnvironmentService extends EnvironmentService {
         this.remoteMachineMetaOccupiedMap.set(remoteEnvironment.rmMachineMeta, false);
     }
 
+    private async getScript(environment: EnvironmentInformation): Promise<string> {
+        const executor = await this.getExecutor(environment.id);
+        const logLevel = getLogLevel();
+        let script: string = environment.command;
+
+        if (executor.isWindows) {
+            const prepare = `mkdir envs\\${environment.id} && cd envs\\${environment.id}`;
+            const startrun = `powershell ..\\install_nni.ps1 && python -m nni.tools.trial_tool.trial_runner`;
+            const developingScript = "IF EXIST nni_trial_tool (ECHO \"nni_trial_tool exists already\") ELSE (mkdir nni_trial_tool && tar -xof ../nni_trial_tool.tar.gz -C ./nni_trial_tool) && pip3 install websockets";
+            
+            script = logLevel == "debug" ? `${prepare} && ${developingScript} && ${startrun}` : `${prepare} && ${startrun}`;
+        }
+
+        script = `cd ${environment.runnerWorkingFolder} && \
```

Comment:
this script value will override line 221.

Reply:
It only works for windows "isdebug" mode or not, Default(Linux) is not affected.

Reply:
FIxed.